.github/ISSUE_TEMPLATE/design_card_template.md

```diff
@@ -1,34 +1,38 @@
 ---
-name: Design Card template
+name: Build Design Card template
```

Comment:
```suggestion
name: Build Design Card template
```
Curious why it says Build Design?

Reply:
The "Build" in front of new design and dev template are a legacy tag when we were trying to separate the documetnation cards even on design and dev from the building one for prototypes purposes. But Im ok removing it , its not needed

Reply:
Do you want to make the suggestion to remove build so I can approve it ?